src/setup/backgroundTask/index.ios.ts

```diff
@@ -0,0 +1,8 @@
+import TaskManager from '@expensify/react-native-background-task';
+import * as SequentialQueue from '@libs/Network/SequentialQueue';
+
+const BACKGROUND_FETCH_TASK = 'FLUSH-SEQUENTIAL-QUEUE-BACKGROUND-FETCH';
+
+TaskManager.defineTask(BACKGROUND_FETCH_TASK, () => {
```

Comment:
Can we add some kind of log here? which can be tracked to Analytics such as Sentry or whatever we're using?

Reply:
Yes, i'd like to also do it - I just need a guidance on what tooling what fit here the best 👍

Reply:
I think we can just leverage `Log.info` here, I see some logic for remote storing these logs.